src/localstorage.js

```diff
@@ -0,0 +1,54 @@
+export default (() => {
+  const sortindex = (todoList) => todoList.map((item, index) => {
+    item.index = index;
+    return item;
+  });
+  // ****Store Tasks in Local Storage //
+  const storetask = (todoList) => {
+    localStorage.setItem('todo-list-group', JSON.stringify(todoList));
+  };
+
+  const gettasks = () => JSON.parse(localStorage.getItem('todo-list-group'));
+  // ****Tasks in the list
+  const add = (event) => {
+    const list = gettasks();
+    list.push({ description: event.target.value, completed: false, index: 0 });
+    storetask(sortindex(list));
+  };
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md) for this project, " `By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).` ", please kindly ensure the index set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5 not 4).